scripts/test/generate-image.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+    cat 1>&2 <<EOF
+Please run with either "debian" or "ubuntu" as argument.
+EOF
+}
+
+if [ "$UID" -ne 0 ]; then
+    echo "We'll need root for this..." 1>&2
+    exec sudo "$0" "$@"
+fi
+
+while [ -n "$1" ]; do
+    case "$1" in
+        "ubuntu")
+            GENERATE_VARIANT=generate_ubuntu
+            ;;
+        "debian")
+            GENERATE_VARIANT=generate_debian
+            ;;
+        *)
+            usage
+            exit 1
+            ;;
+    esac
+    shift
+done
+
+cleanup_losetup() {
+    set +e
+    for dev in ${LO_DEVICE}p*; do
+        umount $dev
+    done
+    losetup -d $LO_DEVICE
+    rmdir tmp-p1
+    rmdir tmp-p2
+}
+
+generate_debian() {
+    local image="Debian-11-x86-64.img"
+
+    mkosi --root-size=2G --distribution=debian --release=bullseye --format=gpt_ext4 --bootable --checksum \
```

Comment:
At the moment I don't really see us supporting more than one, which is already more than enough work. If we start supporting more than one it would make sense to make this configurable.

Reply:
I was thinking more along the lines, at what point will we remember to bump this?